docs/develop/use-polyfills-to-support-ie11-in-your-office-add-in.md

```diff
@@ -0,0 +1,100 @@
+For example, `'planner'.startsWith('plan')` will return `true`.
+The `startsWith()` method is not supported by IE11. So, if you use this method in your script, customers still on IE11 will run into issues. The following code shows how to create a polyfill for this method so your add-in can support customers on IE11.
+
+```javascript
+if (!String.prototype.startsWith) {
+    Object.defineProperty(String.prototype, 'startsWith', {
+        value: function(search, rawPos) {
+            var pos = rawPos > 0 ? rawPos|0 : 0;
+            return this.substring(pos, pos + search.length) === search;
+        }
+    });
+}
+```
+> [!NOTE]
+> For the previous sample, any copyright is dedicated to the Public Domain. [http://creativecommons.org/publicdomain/zero/1.0/](http://creativecommons.org/publicdomain/zero/1.0/).
+
+If your customer is using a current browser that supports the `startsWith()` method, the previous code will detect this, and nothing will change. However, if they’re using IE11 or another legacy browser, the previous code will ensure compatibility and a good customer experience.
+
+## Use an open source library for polyfills
+
+It would be tedious if you had to write individual definitions for every method that might not be supported by IE11. Fortunately, there are open source libraries available that implement the vast majority of necessary polyfills for IE11 and other legacy browsers. The standard at this time is the **core-js** library, as recommended by Microsoft’s documentation for Office Add-ins: [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). Documentation for **core-js** can be found in the [GitHub zloirock/core-js standard library repo](https://github.com/zloirock/core-js).
+After you add the **core-js** library to your Office Add-in project, you can add polyfills for all **core-js** features by importing the library at the top of your entry point. The following example shows how to add the **core-js** library to your script. The `run` function shows an example of calling the `startsWith()` method. The **core-js** library contains numerous polyfills, including the `startsWith()` method.
+
+```javascript
+import "core-js";
+// import "core-js/features/string/starts-with"
+
+Office.onReady((info) => {
+  if (info.host === Office.HostType.Excel) {
+    document.getElementById("sideload-msg").style.display = "none";
+    document.getElementById("app-body").style.display = "flex";
+    document.getElementById("run").onclick = run;
+  }
+});
+
+export async function run() {
+  try {
+      console.log('planner'.startsWith('plan'));
+    } catch (error) {
+    console.error(error);
+  }
+}
+```
+The previous example is based on the [Build an Excel task pane add-in](../quickstarts/excel-quickstart-jquery.md) quick start.
```

Comment:
Consider moving this to before the example so you don't have two consecutive lines that start with "in the previous example."